src/gui/static/src/assets/i18n/cn.json

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
The language code for chinese is zh, not cn. https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

The file should be zh.json